benchmarking/frame-weight-template.hbs

```diff
@@ -89,48 +59,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			{{#each benchmark.component_calculated_proof_size as |cp|}}
 			.saturating_add(Weight::from_parts(0, {{cp.slope}}).saturating_mul({{cp.name}}.into()))
 			{{/each}}
-	}
-	{{/each}}
-}
-
-// For backwards compatibility and tests
-impl WeightInfo for () {
```

Comment:
Are we sure we don't need this?

Reply:
The trait and `unit` implementation should always be defined in the pallet, in `runtime/common/src/weights/*` we want the implementation generated by the benchmark.

https://github.com/moonbeam-foundation/moonbeam/blob/06031d12c369aed299e4c1d0e177b9314e9dd9fe/runtime/common/src/weights/cumulus_pallet_xcmp_queue.rs#L47